zebrad/tests/acceptance.rs

```diff
@@ -8,6 +8,11 @@ use color_eyre::eyre::Result;
 use std::time::Duration;
 use zebra_test::prelude::*;
 
+#[cfg(unix)]
+const KILL_SIGNAL: i32 = 9;
+#[cfg(not(unix))]
+const KILL_SIGNAL: i32 = 1;
```

Comment:
Similarly, let's use generic names, and document:

```suggestion
/// The kill signal exit status on unix platforms.
#[cfg(unix)]
const KILL_STATUS: i32 = 9;
/// The kill exit code on non-unix platforms.
#[cfg(not(unix))]
const KILL_STATUS: i32 = 1;
```